src/main/java/seedu/duke/data/notebook/Note.java

```diff
@@ -10,7 +10,6 @@
     private String title;
     private ArrayList<String> content;
     private Boolean isPinned;
-    private boolean isArchived;
```

Comment:
Why did you remove this variable? I thought it is needed to differentiate the notes when saving to txt file? Or is the archived note list saved in a separate txt file?

Reply:
No need to check for is archived in my implementation. The note will either be in the archived notebook or in the main notebook. When archiving/unarchiving, the note will be removed and added to the respective notebooks.

Reply:
Ya, I am referring to loading from text file, what differentiates them to be loaded into respective array list?

Reply:
How is the files being loaded currently? It traverses notes (from notebook) right? Can just make it traverse both notes and archivedNotes?

Reply:
Take a look at her PR

Reply:
Will add it back in that case